sinch/domains/voice/models/calls/responses.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchBaseModel
+from sinch.domains.voice.models import Price
+
+
+@dataclass
+class GetVoiceCallResponse(SinchBaseModel):
+    from_: str
```

Comment:
Why do you suffix this property with `_` ?

Reply:
`from` is a reserved keyword in Python. Using `_` suffix is a standard python way to deal with this problem.